README.md

```diff
@@ -110,6 +110,18 @@ latest 2 versions of an OS.
 | Android NDK | 1 latest non-LTS, 2 latest LTS versions |
 | Xcode     | - all OS compatible versions side-by-side <br/> - for beta, GM versions - latest beta only <br/> - old patch versions are deprecated in 3 months |
 
+### Package managers usage
+
+There is a number of third-party package managers used during image generation proccess for software installation.
+The table below reflects the list of these PMs and the software installed.

```

Comment:
```suggestion
The table below reflects the list of these package managers and the software installed.

```

Reply:
Could you add a line here as well about how often we review this list of package managers to see if they are still secure?

Reply:
Also any other information we can share about how we check the PMs to ensure they are secure.